doc/mpich/tuning_parameters.md

```diff
@@ -114,3 +114,149 @@ communicate by using different communicator, where each communicator can be
+Network Interface Card (NIC). Notice that PVARS are currently only available through our debug build. 
+The new PVARs to track the amount of bytes sent and received are shown Table V.
+
+**Table V: PVARS to track bytes sent and received**
+
+```
++ ================================ + ================ + ================= + ==== + ======= +
+| Variable Name                    | Class            | Handle            | Type | Storage |
++ ================================ + ================ + ================= + ==== + ======= +
+| nic_sent_bytes_count             | Counter (Array)  | Non-continuous    | SUM  | Static  |
+| nic_recvd_bytes_count            | Counter (Array)  | Non-continuous    | SUM  | Static  |
++ -------------------------------- + ---------------- + ------------------------ + ------- +
+```
+
+The new PVARs to track the amount of data sent using the striping optimization (when messages are 
+sufficiently large) are shown in Table VI. Notice that with striping, an initial handshake and small data 
+transfer is sent via fi_send whereas most of the data transfer takes place through fi_read call.
+Hence, the sum of the two PVARS in Table VI represent the total amount of data transferred through striping.
+
+**Table VI: PVARS to track amount of data sent through striping**
+
+```
++ ================================ + ================ + ================= + ==== + ======= +
+| Variable Name                    | Class            | Handle            | Type | Storage |
++ ================================ + ================ + ================= + ==== + ======= +
+| striped_nic_sent_bytes_count     | Counter (Array)  | Non-continuous    | SUM  | Static  |
+| striped_nic_recvd_bytes_count    | Counter (Array)  | Non-continuous    | SUM  | Static  |
++ -------------------------------- + ---------------- + ------------------------ + ------- +
+```
+
+The new PVARs to track number of bytes sent and received through RMA calls are shown in Table VII.
+
+**Table VII: PVARS to track amount of data sent through RMA calls**
+
+```
++ ================================ + ================ + ================= + ==== + ======= +
+| Variable Name                    | Class            | Handle            | Type | Storage |
++ ================================ + ================ + ================= + ==== + ======= +
+| rma_pref_phy_nic_put_bytes_count | Counter (Array)  | Non-continuous    | SUM  | Static  |
+| rma_pref_phy_nic_get_bytes_count | Counter (Array)  | Non-continuous    | SUM  | Static  |
++ -------------------------------- + ---------------- + ------------------------ + ------- +
+```
+
+Following are the steps to use PVARs in an application:
+
+* Initialize MPI_T interface using `MPI_T_init_thread` routine.
+* Get the number of control variables (PVARs) available in MPICH using `MPI_T_cvar_get_num` routine.
+* Get the information about the control variables (eg., name, class, datatype, etc.) and select the 
```

Comment:
"performance variable" not "control variable", `MPI_T_pvar_get_num` not `MPI_T_cvar_get_num`.